### PR TITLE
Gives them syndie boys some new toys #fuckthefreeze

### PR DESCRIPTION
**Pull request by kilkun**
[Feature Freeze!]: # We are currently not considering any balance or antagonist oriented pull requests. Full details, as well as ways to bypass this freeze, are available here https://github.com/tgstation/tgstation/pull/28223

hol up. I'm real happy for you and your feature freeze and I'mma let you resume it in just a sec.

:cl:
experiment: Reports are coming in of Syndicate strike teams deploying experimental super soldiers. Crew are advised to be on alert.
/:cl:

This PR finally puts the corporate species datum into use. Basically, for 90TC (to be tweaked), Ops can buy a modified implanting chamber that will turn one of them into a combat cyborg. Cyborg as in the traditional idea, not the in-game established cyborg race. They also get a swanky space-proof greatcoat, since the whole thing is a reference to Bullfrog's Syndicate agents of the 1993 cult classic, Syndicate.

Also it puts the officer's jacket into use. Previously it was used for a long-deprecated equipment datum. I simply debuffed it's armor values and made it use the greatcoat sprite.

I'm sure this will get closed very soon because it violates the feature freeze but gosh dangit as soon as it's over you can be sure this PR is coming back!

### CODE REVIEW

**Comment by MoreRobustThanYou** [comment]
I like this. Speedmerge please!

**Comment by Repukan** [comment]
Freeze

**Comment by Repukan** [comment]
Also aren't combat cyborgs the ones with no attack delay

**Comment by kilkun** [comment]
no

**Comment by ChangelingRain** [comment]
it's a freeze, my boy

**Comment by kilkun** [comment]
YOU HAVE TO COMMENT ON THE IMPLEMENTATION

YOU PROMISED

File: code/game/objects/items/weapons/implants/implantchair.dm [hunk 1]
```diff
@@ -187,3 +187,38 @@
...
+			visible_message("<span_class='warning'>The machine buzzes. It will only operate on a naked human.</span>")
+			playsound(get_turf(src), 'sound/machines/buzz-two.ogg', 50, 1)
+			return 0
+		else
+			visible_message("<span class='warning'>A horrible screech and a bright flash of light eminate from the [src]. In the aftermath, cold fog flows from the chamber, pooling from the floor.</span>")
+			playsound(get_turf(src), 'sound/effects/screech.ogg', 50, 1)
+			to_chat(H, "<span class='heavy_alloy'>Your body is twisted in terrible trauma as robotic arms amputate appendages and restore them with resplendent replacements. You feel a sharp pain in your head and your mind shatters as you lose your concept of self. The agony leaves you as soon as it came and you feel much lighter without your humanity weighing you down.</span>")
+			to_chat(H, "<span class='heavy_alloy'>You feel leather and cloth wrap around your body, armor fit for your new form.</span>")
+			H.equipOutfit(/datum/outfit/syndicate_agent)//Gives them a space suit + gloves + headset + boots. no helmet though.
+			human.set_species(/datum/species/corporate)
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/implants/implantchair.dm]: ah, yes, 20-30 damage punches, on top of, generally resistant to brute and burn(30% and 35% reductions), being immune to cold damage, resistant to heat damage(50% reduction on top of the burn reduction), having super sanic speed(a speedmod of -2), being immune to radiation and viruses, having no blood, feeling no hunger, being impossible to dismember, and not being affected by stepping on glass.

Comment (kilkun) [inline code/game/objects/items/weapons/implants/implantchair.dm]: 90 telecrystals gets you one. You can literally buy this or a gygax and an energy sword and two briefcases full of cash.

**Comment by kilkun** [comment]
in 90% of cases that gygax will probably be more useful, as it can't be stunned and is a lot tougher.

this however is the final word in stealth ops. it can blend in decently well (a lot better than a mech) without sacrificing the ability to hit like a train if shit gets hot.

arguably it's less useful than if you gave an operative a hulk mutation. at least a hulk can't get stunned and lasered to death.

**Pull request closed without merging**